src/workloads/sharding/WriteOneWithoutShardKeyUnshardedCollection.yml

```diff
@@ -0,0 +1,119 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+Description: |
+  On an unsharded collection on a single shard cluster, run workloads that updateOne, deleteOne, and findAndModifyOne.
+
+GlobalDefaults:
+  Nop: &Nop {Nop: true}
+
+  Database: &Database test
+  UnshardedCollection: &UnshardedCollection Collection1
+
+  DocumentCount: &DocumentCount 10000  # Number of documents to insert and modify.
+  BalancerWait: &BalancerWait "20 seconds"  # Wait out the 20 second delay between balancing rounds
```

Comment:
Same in this workload, the balancer isn't necessary and will needlessly slow down the workload.